.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+Resolves #{PUT_ISSUE_NUMBER_HERE}
+
+<!-- You can add any comments here -->
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+
+### General
```

Comment:
I think that `### General` section can be removed to make the checklist look smaller. Since we have only single section. @vrom911, what do you think?